rules/windows/malware/win_mal_ryuk.yml

```diff
@@ -11,10 +11,14 @@ logsource:
     product: windows
 detection:
     selection:
-        CommandLine:
-            - '*\net.exe stop "samss" *'
-            - '*\net.exe stop "audioendpointbuilder" *'
-            - '*\net.exe stop "unistoresvc_?????" *'
+        ProcessName|contains:
+            - 'net.exe'
+        CommandLine|contains:
+            - 'stop'
+        CommandLine|contains:
```

Comment:
I believe that double `CommandLine|contains:` is gonna break the logic. Have you tried to convert it locally?

Reply:
Sorry, missed the `|all`, i think it maintains the logic now